Makefile

```diff
@@ -209,9 +209,21 @@ gitserver:
 		-t $(GITSERVER_IMAGE) \
 		--push .
 
+.PHONY: e2e-test-clean
+e2e-test-clean:
+	$(KUBECTL) delete -k tests/e2e/manifests --ignore-not-found=true
+	$(KUBECTL) delete -f tests/e2e/manifests/numaplane-ns.yaml --ignore-not-found=true
+
+.PHONY: e2e-test-start
+e2e-test-start: e2e-test-clean image
+	$(KUBECTL) apply -f tests/e2e/manifests/numaplane-ns.yaml
```

Comment:
We should also add a step to delete and create `numaflow-pipeline` namespace as that is the current target namespace for GitSync e2e testing. We could consider making this variable in the future.